lib/github/errors.rb

```diff
@@ -8,7 +8,7 @@ class << self
       def with_error_handling
         yield
       rescue Octokit::Error => err
-        Failbot.report!($ERROR_INFO)
+        Failbot.report!($ERROR_INFO) unless err.class == Octokit::NotFound
```

Comment:
LGTM 👍 why are we getting 404s in the first place?

Reply:
We get TONS of 404s. They happen mostly when a resource on our end no longer exists on GitHub.com